packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -452,21 +452,26 @@ abstract class BoxScrollView extends ScrollView {
 /// machinery can make use of the foreknowledge of the children's extent to save
 /// work, for example when the scroll position changes drastically.
 ///
-/// There are three options for constructing a [ListView]:
+/// There are four options for constructing a [ListView]:
```

Comment:
(you don't have to make this change, but fwiw, this diff i why i usually say "There are several options...")